tests/phpUnit/Api/Projects/GetRecommendedTest.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Tests\phpUnit\Api\Projects;
+
+use App\Api\ProjectsApi;
+use App\Entity\Program;
+use App\Entity\ProgramManager;
+use App\Entity\User;
+use App\Utils\ElapsedTimeStringFormatter;
+use OpenAPI\Server\Model\ProjectResponse;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use ReflectionException;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Tests\phpUnit\TestUtils\PHPUnitUtils;
+
+/**
```

Comment:
Annotations could be more specific. What is our use case for @internal?

 - @group unit or integration,
 - @cover ::method (unit)